src/app/report/report.component.html

```diff
@@ -12,7 +12,8 @@
       <mat-icon style="color: grey;">close</mat-icon>
     </button>
   </div>
-  <mat-tab-group mat-stretch-tabs class="tabs">
+  <embed style="margin: 0 auto;" src="assets/loading.svg" class="loadingSVG" [style.display]="isVisible ? 'none': 'block'" />
```

Comment:
You might wanna not use margins here. You play around with a few CSS properties. See what happens when you use on the parent div,
```css
display: flex;
justify-content: center;
align-items: center;
flex-direction: column;
```

Try it out! You want to try to make it as responsive as possible!